lnpbp-0021.md

```diff
@@ -56,102 +57,113 @@ Interface specification is the following Contractum code:
 ```haskell
 -- # Defining main data structures
 
--- collectibles are usually scarse, so we limit their max number to 64k
 data ItemsCount :: U32
```

Comment:
Hi @dr-orlovsky 

I started to implement the interface, but I missed this data type in the interface. This type of data [and to reference the total amount of items that can be issued?

Reply:
You just need to write
```rust
pub struct ItemsCount(u32);
```
in rust and do all derives on this type (for strict types, wrapper, arithmetics, display, fromstr etc) - and than compile it as a part of the type library used by the interface.

I will do that, don't worry - I have to do the implementation of all these interfaces anyway